.github/workflows/publish-fig-spec.yml

```diff
@@ -0,0 +1,54 @@
+name: Publish Fig Completion Spec
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
```

Comment:
Is this trigger just for testing? As the workflow is written I'm not sure it makes sense to run in PRs

```suggestion
```